### DISCUSSION TITLE
El-Upload not marking files as uploaded when manually added to filelist

### DISCUSSION THREAD

**Discussion by michaeltoohig** [element-plus/element-plus#10798] [Q&A]
I'm using the el-upload `addFile` method to add an image that the user modifies in the fileList. Upon calling the `submit` method the el-upload component does not display the green mark on the top right corner of the image that it was successfully uploaded. This means repeatedly calling the `submit` method will upload the file each time. When the user does not modify the image, and therefore doesn't manually add an image into the fileList the file is uploaded and marked as uploaded correctly and further attempts at `submit` will not upload to server again.

Is there a documented way to modify the fileList correctly?

Thank you.

```
<template>
<Upload ref="uploadRef" :action="action" v-on:on-preview="onPreview" />
</template>

<script setup>
const uploadRef = ref<UploadInstance>();

const onCrop = (data: blob) => {
  uploadRef.value!.addFile(data);
}
</script>
```

**Comment by chenxch** [comment]
Is it convenient to use the playground to provide a simple demo, which is a bit difficult to solve the problem.
